src/handlers.js

```diff
@@ -110,18 +110,31 @@ async function homePage(req, res, timecop) {
   if (cached !== null) {
     timecop.end("cache-check");
     // We know our cache is good and lets serve the data
-    res.render("home", { featured: cached, timecop: timecop.timetable, page: homePage });
+    res.render("home", { ...cached, timecop: timecop.timetable, page: homePage });
```

Comment:
I'll also want to be aboslutely sure that the cache still functions with both options. While it should, just because it's written as JSON, I'd want to make sure the two objects are parsed as JSON text properly

Reply:
Sounds good. Yeah, theoretically it should work, but I am not familiar with Google storage to test. There may be a hiccup on the first initialization where the cache does not match the new `{featuredPackages: [], featuredThemes: []}` object.

Reply:
Yeah good point. There shouldn't be, but after deployment I'll go ahead and manually delete the cache just in case.